.circleci/upload-docs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+USERNAME=${CIRCLE_PROJECT_USERNAME}
+REPONAME=${CIRCLE_PROJECT_REPONAME}
+PR_NUM=${CIRCLE_PR_NUMBER}
+
+if [ -n "${PR_NUM}" ] # If build is a PR
+then
+  # Domain names follow the RFC1123 spec [a-Z] [0-9] [-] limited to 253 characters
+  # https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax
+  # So, just replace "/" or "." with "-"
+  DEPLOY_SUBDOMAIN=`echo "${REPONAME}-pr-$PR_NUM" | tr '[\/|\.]' '-' | cut -c1-253`
+  ALREADY_DEPLOYED=`npx surge list | grep ${DEPLOY_SUBDOMAIN}`
+else
+  DEPLOY_SUBDOMAIN=${REPONAME}
+fi
+
+DEPLOY_DOMAIN="https://${DEPLOY_SUBDOMAIN}.surge.sh"
+npx surge --project public --domain $DEPLOY_DOMAIN;
+
+if [ -n "${PR_NUM}" ] && [ -z "${ALREADY_DEPLOYED}" ] # Leave a Github comment
+then
+  # Use Issues api instead of PR api because
+  # PR api requires comments be made on specific files of specific commits
+  GITHUB_PR_COMMENTS="https://api.github.com/repos/${USERNAME}/${REPONAME}/issues/${PR_NUM}/comments"
+  echo "Adding github PR comment ${GITHUB_PR_COMMENTS}"
+  curl -H "Authorization: token ${GH_PR_TOKEN}" --request POST ${GITHUB_PR_COMMENTS} --data '{"body":"PatternFly-Next preview: '${DEPLOY_DOMAIN}'"}'
```

Comment:
I wonder why we're seeing PatternFly-_React_ preview instead of PatternFly-Next preview? in the github comments?

<img width="788" alt="Screen Shot 2019-08-02 at 2 24 14 PM" src="https://user-images.githubusercontent.com/5942899/62390875-570cea80-b531-11e9-8ae8-7f5fd226e5f2.png">

Reply:
That's because I didn't fix that script until a few commits in. The next one will correctly say PatternFly-Next preview.

Reply:
I see, thx. Can you rebase this branch so we can verify it in action before merging?